README.md

```diff
@@ -26,10 +26,12 @@ If you are using Maven with a BOM, add this to your pom.xml file.
  </dependencies>
 </dependencyManagement>
 
-<dependency>
-  <groupId>com.google.cloud</groupId>
-  <artifactId>google-cloud-monitoring</artifactId>
-</dependency>
+<dependencies>
```

Comment:
while we're here, want to update the libraries-bom version above? currently at 2.8.0

Reply:
Sorry, just saw the other PR you raised. Let's combine into one?

Reply:
done